src/Header.js

```diff
@@ -8,54 +8,31 @@ const Header = () => {
   return (
     <header className="Header">
       <div className="title-container">
-        <h1 className="title">
-          WRITTEN <span>IN THE</span> STARS
-        </h1>
+        <h1 className="title">WRITTEN IN THE STARS</h1>
+      </div>
+      <div className="nav-buttons">
+        <DropdownButton
+          className="drop-button"
+          id="dropdown-basic-button"
+          title="ZODIAC SIGNS"
+        >
+          <Dropdown.Item href="/aries">Aries</Dropdown.Item>
```

Comment:
The dropdown menu functions as it should, cool importing bootstrap and a css file